lib/file/index.js

```diff
@@ -70,18 +71,34 @@ const outputSQLFiles = (xlsxContent) => {
   const distinctTableList = distinct(dataDict.map((x) => x['Table Name'])).filter((x) => x && x !== '');
 
   distinctTableList.forEach((table) => {
+    const selectComment = SqlComment.gen("SELECT", `select-${table}`);
     const selectStatement = getSelectStatement(table, dataDict);
+    const executableSelectSql = selectComment+SqlComment.reverse(selectStatement);
```

Comment:
Suggest to use template string `${selectComment}${SqlComment.reverse(selectStatement)}` instead of +